### PR TITLE
1.19.2 forge

### PR DESCRIPTION
**Pull request by Mysticpasta1**
this is meant for a 1.19.2 forge branch if you need me to re PR I will :D

### CODE REVIEW

**Comment by Mysticpasta1** [comment]
btw the 1.19.2 commit is because intellij broke and deleted 3 - 4 hours of work on one of the class

File: build.gradle [hunk 1]
```diff
@@ -15,7 +15,7 @@ apply plugin: 'org.parchmentmc.librarian.forgegradle'
 apply plugin: 'eclipse'
 apply plugin: 'maven-publish'
 
-version = '0.10.2b'
+version = '1.0.0'
```
Comment (hypherionmc) [inline build.gradle:18]: Please don't bump to 1.0.0. Bump to 0.11.0 instead

File: src/main/java/kirothebluefox/moblocks/content/ModBlocks.java
Comment (hypherionmc) [inline src/main/java/kirothebluefox/moblocks/content/ModBlocks.java:1]: Are you sure this file is no longer needed?

Pretty sure these references were set dynamically by forge using `@ObjectHolder`. Same for the other classes you deleted

**Review by hypherionmc — CHANGES_REQUESTED** [review]
Just some changes. Also please edit your commit message. This mod doesn't belong to either of us and I don't think Kiro would like swearing in the commits

File: src/main/java/kirothebluefox/moblocks/content/ModBlocks.java
Comment (Mysticpasta1) [reply src/main/java/kirothebluefox/moblocks/content/ModBlocks.java:1]: The `@ObjectHolder` annotation is removed on 1.19.2 forge and should be replaced in your registry class with `DefferedRegistry` or whatever it is called now. Also due to intellij wiping out 3 - 4 hours of work I just left the giant registry class alone, which will need to be fully rewritten!

**Comment by Mysticpasta1** [comment]
I fixed the swearing on that commit I was just angry at intellij deleting 3 - 4 hours of work

File: src/main/java/kirothebluefox/moblocks/content/ModBlocks.java
Comment (hypherionmc) [reply src/main/java/kirothebluefox/moblocks/content/ModBlocks.java:1]: Fair enough. Yeah as I mentioned to you in discord, major parts of the mod need to be rewritten, especially if we plan on going above 1.19.2, but I'll speak with you then we can tackle that later on.

For now, the PR looks good

**Pull request merged**